surveys/2024-annual-survey/questions.md

```diff
@@ -376,7 +376,7 @@ Type: free text
 Type: matrix (optional)
 
 - Implementing logic for tuples of various sizes
-- Splitting code across crates (orphan rule)
+- Splitting code across crates (e.g. orphan rule)
```

Comment:
I usually use these shortcuts ("e.g.", "i.e.") but I wonder if it's a slight barrier for some non-English people filling this survey.

I have thought of suggesting a massive search and replace for "e.g." with "for example" and for "i.e." with "[...] that is [...]" (or other wording) but I am not brave enough to push such a proposal :sweat_smile: